HorizonEditor/src/EditorLayer.cpp

```diff
@@ -48,37 +48,31 @@ void EditorLayer::onDetach()
 
 void EditorLayer::onUpdate(Hzn::TimeStep ts)
 {
-	if (m_ViewportFocused && m_ViewportHovered && !m_PlayMode) {
-		m_EditorCameraController.onUpdate(ts);
-	}
-
 	m_FrameBuffer->bind();
 
 	// here, in case if the framebuffer is re-created, and the last known
 	// viewport size does not match the viewport size of the new framebuffer, then
 	// we update all the camera components to the proper aspect ratio, and update the last known viewport size.
 	auto& props = m_FrameBuffer->getProps();
-
-	if (EditorData::s_Scene_Active) {
-		lastViewportSize = EditorData::s_Scene_Active->onViewportResize(props.width, props.height);
+	if (Hzn::SceneManager::isOpen()) {
```

Comment:
Is the last viewport size stored in memory?

Reply:
Yeah to re-create frame buffers, Update Cameras, to align with the aspect ratio.